ArduPlane/tailsitter.cpp

```diff
@@ -62,6 +62,139 @@ void QuadPlane::tailsitter_output(void)
     plane.pitchController.reset_I();
     plane.rollController.reset_I();
 
+    // scale factor required for normalizing roll/pitch/yaw servo values
+    const float angle_servo_max = SERVO_MAX;
+    const float throttle_servo_max = 100.0f;
+
+    static float orig_pkP = -1.0f;
+    static float orig_rkP = -1.0f;
+    static float orig_ykP = -1.0f;
+
+    if (tailsitter.tcomp_gain > 0) {
+        if (orig_pkP < 0.0f) {
```

Comment:
I think the big lump of code here could be moved to a new function, maybe tailsitter_tcomp() ?